cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -585,14 +589,14 @@ func (dc *DownstreamController) syncMissions() {
 
 			// send to all nodes
 			dc.lc.EdgeClusters.Range(func(key interface{}, value interface{}) bool {
-				nodeName, ok := key.(string)
+				clusterName, ok := key.(string)
 				if !ok {
 					klog.Warning("Failed to assert key to sting")
 					return true
 				}
 				msg := model.NewMessage("")
 				msg.SetResourceVersion(mission.ResourceVersion)
-				resource, err := messagelayer.BuildResource(nodeName, "default", model.ResourceTypeMission, mission.Name)
+				resource, err := messagelayer.BuildResource(clusterName, "default", model.ResourceTypeMission, mission.Name)
```

Comment:
Is "default" the model namespace?

Reply:
Actually, this is a workaround. The work to build a cluster-scope resource, where the namespace should be "", is not done. So the message will not be received if the namespace is "". Yet the message will be correctly received as long as both sides have the same namespace name. So I just put it as "default" in the PoC work.